web_src/js/components/RepoActionView.vue

```diff
@@ -193,6 +197,11 @@ const sfc = {
       await this.fetchPost(`${jobLink}/rerun`);
       window.location.href = jobLink;
     },
+    // rerun workflow
+    rerun() {
+      this.fetchPost(`${this.run.link}/rerun`);
```

Comment:
```suggestion
      await this.fetchPost(`${this.run.link}/rerun`);
```

Please test it. Ideally it would also have error handling, but I guess this kind of depends on https://github.com/go-gitea/gitea/issues/24353.